test/functional/index.test.js

```diff
@@ -87,13 +89,10 @@ describe('Indexes', function() {
             // Create an index on the collection
             db.createIndex(
               collection.collectionName,
-              [
-                ['a', -1],
-                ['b', 1],
-                ['c', -1]
-              ],
+              { a: -1, b: 1, c: -1 },
```

Comment:
Same here; I don't see it documented that `createIndex` should work with that syntax. Is it something we need to support?

Reply:
I would assume if this is how it worked in the past, we'd need to keep it.